unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BuilderProjectsPanel/Scripts/Views/MenuSections/SectionDeployedScenesView.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using DCL.Helpers;
+using UnityEngine;
+using UnityEngine.UI;
+
+internal class SectionDeployedScenesView : MonoBehaviour
```

Comment:
Related to the pervious suggestion: I think doing an interface for this class would ease the testing for its consummers (that would use NSubstitute for the assertion).